e2e_tests/tests/test_system.py

```diff
@@ -92,6 +92,12 @@ def test_gc_checkpoints_s3(secrets: Dict[str, str]) -> None:
     run_gc_checkpoints_test(config)
 
 
+@pytest.mark.e2e_gpu  # type: ignore
+def test_gc_checkpoints_s3_prefix(secrets: Dict[str, str]) -> None:
```

Comment:
again, just parameterize the existing test.

Reply:
done